koap/src/jvmMain/kotlin/DebugJvm.kt

```diff
@@ -0,0 +1,4 @@
+package com.juul.koap
```

Comment:
nitpick: I prefer `Debug.kt` filename (to complement the `Debug.kt` filename on the common side).

```suggestion
@file:JvmName("DebugJvm")

package com.juul.koap
```